Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h

```diff
@@ -308,6 +310,7 @@ face_border_length(typename boost::graph_traits<PolygonMesh>::halfedge_descripto
   *   - `first`: a halfedge on the longest border.
   *     The return type `halfedge_descriptor` is a halfedge descriptor. It is
   *     deduced from the graph traits corresponding to the type `PolygonMesh`.
+  *     `first` and it is among the halfedges reported by `extract_boundary_cycles()`.
```

Comment:
A bit a strange grammar.

Reply:
The "and"  is too much.  Without, do you find the sentence still strange?

Reply:
I think that would be better.

Reply:
The fix with `and it` removed will also do.

Reply:
Not really, because the first half of the sentence is about the type and not about the variable.